ruff.toml

```diff
@@ -50,6 +50,12 @@ unfixable = []
 # Allow unused variables when underscore-prefixed.
 dummy-variable-rgx = "^(_+|(_+[a-zA-Z0-9_]*[a-zA-Z0-9]+?))$"
 
+[lint.isort]
+# Ruff sometimes mistakenly thinks the wandb python module is "first-party"
+# and puts it in that section  of the imports if the wandb cache directory
+# exists. See https://github.com/astral-sh/ruff/issues/10519
+known-third-party = ["wandb"]
```

Comment:
This has happened a bunch - we've been bouncing the wandb import up and down the import list because if the "wandb" cache directory and thinks that means it's a "first-party" package instead of "third-party"